### DISCUSSION TITLE
Setting Gyro Offset After Adding Vision Measurement

### DISCUSSION THREAD

**Discussion by silverlightning926** [BroncBotz3481/YAGSL-Example#123] [Q&A] [answered]
A bit confused about the recommendation here on the documentation for `SwerveDrive.addVisionMeasurment`:
<img width="487" alt="image" src="https://github.com/BroncBotz3481/YAGSL-Example/assets/38925407/58ab4e2a-048c-48d1-a916-aecfceb24029">

It recommends that I update my gyro offset myself after I use `SwerveDrive.addVisionMeasurment`, although when looking add the actual method `SwerveDrive.addVisionMeasurment`, there seems to be [a line in there setting the gyro offset already](https://github.com/BroncBotz3481/YAGSL/blob/3fd8493ccbbd0cc274aaac029d39d9626f78d82b/swervelib/SwerveDrive.java#L982) so not sure if I need to set it myself again?

Internal `SwerveDrive.addVisionMeasurment` method:
<img width="585" alt="image" src="https://github.com/BroncBotz3481/YAGSL-Example/assets/38925407/1c864ab7-84e8-45d4-aca7-4d5954c78e61">

Is this just a symptom of the documentation needing to be updated?

**Comment by thenetworkgrinch** [comment, marked as answer]
I fixed this earlier, sorry!
